internal/util/csiconfig.go

```diff
@@ -46,6 +48,7 @@ type ClusterInfo struct {
 // [
 // 	{
 // 		"clusterID": "<cluster-id>",
+//		"namespace": "<namespace>",
```

Comment:
use `"radosNamespace"` in the comment too please

Reply:
Sorry, I missed it :)) it's now fixed.

Reply:
this comment still mentions `"namespace"`, not `"radosNamespace"` as it is used everywhere else

Reply:
ah, it gets addressed in the next commit :+1: